crates/volta-core/src/tool/npm/resolve.rs

```diff
@@ -41,16 +36,7 @@ fn fetch_npm_index(hooks: Option<&ToolHooks<Npm>>) -> Fallible<(String, PackageI
         _ => public_registry_index("npm"),
     };
 
-    let spinner = progress_spinner(format!("Fetching public registry: {}", url));
-    let metadata: RawPackageMetadata = attohttpc::get(&url)
-        .header(ACCEPT, NPM_ABBREVIATED_ACCEPT_HEADER)
-        .send()
-        .and_then(Response::error_for_status)
-        .and_then(Response::json)
-        .with_context(registry_fetch_error("npm", &url))?;
-
-    spinner.finish_and_clear();
-    Ok((url, metadata.into()))
+    fetch_public_index(url, "Npm")
```

Comment:
Super tiny nit: My understanding is that `npm` should always be lowercase as it's part of their brand.

```suggestion
    fetch_public_index(url, "npm")
```

Reply:
Oh oops, didn't realize I changed the casing of that error. Good catch!